spec/controllers/api/exchange_products_controller_spec.rb

```diff
@@ -51,12 +51,27 @@ module Api
         let(:exchange) { order_cycle.exchanges.outgoing.first }
         let(:products_relation) { Spree::Product.includes(:variants).where("spree_variants.id": exchange.variants.map(&:id)) }
 
-        it "paginates results" do
-          spree_get :index, exchange_id: exchange.id, page: 1, per_page: 1
+        before do
+          stub_const("Api::ExchangeProductsController::DEFAULT_PER_PAGE", 1)
```

Comment:
`stub_const` is legacy and we should use:

```suggestion
          stub_const("Api::ExchangeProductsController::DEFAULT_PER_PAGE", 1)
          allow(
```